dev/Repeater/InspectingDataSource.h

```diff
@@ -24,6 +24,7 @@ class InspectingDataSource :
 #pragma endregion
 
     int IndexOf(winrt::IInspectable const& value);
```

Comment:
I think this is not used anymore.

Reply:
IndexOf is still being used as we currently rely on casting unfortunately until the API is public.

Reply:
Ah right :) can you please add a comment that it can be removed once the API is public ?